tests/api.c

```diff
@@ -5164,6 +5182,9 @@ typedef int (*cbType)(WOLFSSL_CTX *ctx, WOLFSSL *ssl);
 
 static int test_client_nofail(void* args, cbType cb)
 {
+    (void)args;
```

Comment:
These must be below the variable declaration:

```
[all-gcc-c99] [4 of 15] [ee684d0d89]
    configure...   real 0m16.121s  user 0m11.501s  sys 0m6.456s
    build...In file included from tests/api.c:134:
tests/api.c: In function ‘test_client_nofail’:
./wolfssl/test.h:169:22: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
  169 |     #define SOCKET_T int
      |                      ^~~
tests/api.c:5188:5: note: in expansion of macro ‘SOCKET_T’
 5188 |     SOCKET_T sockfd = 0;
      |     ^~~~~~~~
```

Reply:
Fixed them.